delivery_ups_oca/tests/test_delivery_ups.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2020 Hunki Enterprises BV
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests import Form, common
+
+
+class DeliveryUps(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        product_shipping_cost = cls.env["product.product"].create(
+            {
+                "type": "service",
+                "name": "Shipping costs",
+                "standard_price": 10,
+                "list_price": 100,
+            }
+        )
+        cls.carrier = cls.env["delivery.carrier"].create(
+            {
+                "name": "UPS",
+                "delivery_type": "ups",
+                "product_id": product_shipping_cost.id,
+                "price_method": "fixed",
+                "ups_default_packaging_id": cls.env.ref(
+                    "delivery_ups_oca.product_packaging_ups"
+                ).id,
+                "ups_ws_username": "ups_odoo_test",
```

Comment:
Put a comment here saying that this account was issued by Odoo S.A. and we are borrowing it from data seen on enterprise runbot.